docker/dotnet/utils/dotnet-yamldotnet.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+
+set -x
+
+# to suppress "Welcome to .NET.." FTUE message
+touch "$HOME"/.dotnet/"$(dotnet --version)".dotnetFirstUseSentinel
+
+build () {
+    dotnet publish /buildutils/"$1" \
+        --nologo                    \
+        --output /build/bin         \
+        --configuration Release     \
+        --runtime linux-musl-x64
+
+  # change permission from 744 to 755 (otherwise we
+  # get "permission denied" in runtime container)
+  chmod 755 /build/bin/*$1
```

Comment:
Thank you. I had to add this workaround as the produced binary by dotnet (1) is strict with permissions: `744` (octal). OTOH, clang/gcc for both C/C++ programs produces binary with <code>7<ins>7</ins>5</code> permission. Our runtime container at least needs `754`, as in, the Group Permission should be Read+Execute (just found out that I should have used `754` for least modification).